### DISCUSSION TITLE
tt.create_session()

### DISCUSSION THREAD

**Discussion by DrEmmanuel7** [atoti/atoti#796] [Q&A] [answered]
When I import atoti and try to apply this code:
Session= tt. Create_session() or
Session = tt. session () I get respectively an AttributeError and an TypeError(module object is callable). 
I need your help.

**Comment by HuifangYeo** [comment, marked as answer]
Hi,

Session should be uppercased, i.e. :

`session = tt.Session()`

Kindly refer to the [Atoti tutorial](https://docs.atoti.io/latest/getting_started/tutorial/tutorial.html#From-CSV-to-Cube).
